hack/create_mock_bundles.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -eou pipefail
+
+# Define the path to the test resources directory relative to the script location
+TEST_RESOURCES_DIR="$(dirname "$0")/../test/resources"
+
+# Ensure the test resources directory exists
+mkdir -p "$TEST_RESOURCES_DIR"
+
+# Function to create a bundle directory with subdirectories
+create_bundle() {
+    local BUNDLE_DIR="$TEST_RESOURCES_DIR/$1"
+    echo "Creating $BUNDLE_DIR..."
+
+    # Remove the bundle directory if it already exists
+    if [ -d "$BUNDLE_DIR" ]; then
+        echo "Removing existing directory: $BUNDLE_DIR"
+        rm -rf "$BUNDLE_DIR"
+    fi
+
+    mkdir -p "$BUNDLE_DIR"
+
+    # Shift to process subdirectories
+    shift
+    for SUBDIR in "$@"; do
+        local FULL_PATH="$BUNDLE_DIR/$SUBDIR"
+        mkdir -p "$FULL_PATH"
+        touch "$FULL_PATH/.gitkeep"
+        echo "Created $FULL_PATH with .gitkeep"
+    done
+}
+
+# Create the bundle directories and subdirectories
+create_bundle "bundle-1" "etc" "usr" "usr/lib64"
```

Comment:
This is maybe a nit, but this seems to create a mock unpacked image with /etc/, /usr, /usr/lib64. 

Use of the term "bundle" confused me a bit because I think of https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md

which is a stripped down image whose only content is metadata in the /manifests and /metadata directories (and maybe a couple other directories).

Reply:
yeah I can see that confusion. changing the naming to "mock unpacked filesystem" in d21f5cdc